modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -34,111 +51,296 @@ Or:
+
+Asset Viewer:: Enables a user to view and download an asset. 
++
+Users with this permission cannot edit asset portal content or share an asset with another user. 
++
+Use this permission to invite a user outside your business group to view and download an asset.
+
+Asset Contributor:: Enables a user to view, add a new version, and download an asset. 
++
+Use this permission to invite a user outside your business group to view, download, and add edit portal content for an asset.
+
+Asset Administrator:: Enables a user to view, create, download, deprecate, and delete an asset. 
++
+Users with this permission have the same access as users with the Exchange Administrator permission, but on only a single asset. 
++
+This permission is assigned by default to an asset creator. 
++
+Use this permission to extend administrator permissions to an asset to an additional user.
+
+== Monitoring
+
+Monitoring Administrator:: Enables a user to view, create, modify, and delete content in Anypoint Monitoring.
+
+Monitoring Viewer:: Enables a user to view but not modify content in Anypoint Monitoring.
+
+== MQ
+
+View clients:: Enables a user to view all client apps, including client app IDs and client secrets for each client app.
+
+View destinations:: Enables a user to:
++
+* View all destinations and each destination’s settings (ID, Type, Message TTL, and Message Lock Default TTL).
+* View In Queue messages. 
+* View In Flight message stats.
+
+Clear destinations:: Enables a user to: 
++
+* View all destinations and each destination’s settings (ID, Type, Message TTL, and Message Lock Default TTL).
+* View In Queue messages.
+* View In Flight message stats.
+* Clear destinations.
+
+Manage clients:: Enables a user to: 
++
+* View all client apps, including client app IDs and client secrets for each client app.
+* Create client apps.
+
+Administer destinations:: Enables a user to:
++
+* View all destinations and each destination’s settings (ID, Type, Message TTL, and Message Lock Default TTL).
+* View In Queue messages.
+* View In Flight message stats.
+* Clear destinations.
+* Create new queues, message exchanges, and bindings.
+* Edit existing queues, message exchanges, and bindings.
+* Purge messages from queues.
+
+Manage destinations (deprecated):: Enables a user to: 
++ 
+* View all destinations and each destination’s settings (ID, Type, Message TTL, and Message Lock Default TTL).
+* View In Queue messages.
+* View In Flight message stats.
+* Clear destinations.
+* Create new queues, message exchanges, and bindings.
+* Edit existing queues, message exchanges, and bindings.
+* Purge messages from queues.
+* Edit existing queues and message exchange settings.
+* Access Message Sender and Browser pages.
+* Delete messages.
++
+This permission is deprecated. To achieve the same abilities as Manage permissions, assign these permissions to the user: 
```

Comment:
This is confusing. All of the permissions in the list are deprecated, so why is this one signaled out? Also, why is the extra text here?